packages/blade/src/components/Popover/types.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { Side, UseFloatingOptions } from '@floating-ui/react';
+import type { CSSProperties } from 'react';
+import type React from 'react';
+import type { GestureResponderEvent } from 'react-native';
+import type { BaseBoxProps } from '~components/Box/BaseBox';
+import type { Platform } from '~utils';
+
+type PopoverProps = {
+  title?: string;
+  titleLeading?: React.ReactNode;
+  footer?: React.ReactNode;
+  /**
+   * Popover content
+   */
+  content: React.ReactElement;
+  /**
+   * Placement of Popover
+   *
+   * @default "top"
+   */
+  placement?: UseFloatingOptions['placement'];
+  children: React.ReactElement;
+  isOpen?: boolean;
+  defaultIsOpen?: boolean;
+  onOpenChange?: ({ isOpen }: { isOpen: boolean }) => void;
+  /**
+   * Sets the z-index of the modal
+   * @default 1100
+   */
+  zIndex?: number;
+  initialFocusRef?: React.RefObject<any>;
+};
+
+type PopoverContentProps = {
+  title?: string;
+  titleLeading?: React.ReactNode;
```

Comment:
Because we forward those props in PopoverContent, PopoverContent is an internal component. 

I guess i can do a pick instead `Pick<PopoverProps, 'title' | 'titleLeading' | 'footer' | 'children'>`